woodwork/tests/type_system/conftest.py

```diff
@@ -332,6 +332,27 @@ def pdnas(request):
     return request.getfixturevalue(request.param)
 
 
+# Empty Series Inference Fixtures
+@pytest.fixture
+def pandas_empty():
```

Comment:
Minor nitpick, but I would maybe call this `pandas_empty_series` to make it clear this is a series and not a dataframe like most of the other fixtures.